### ISSUE TITLE
Add delta percentage?

### ISSUE THREAD

**Issue by mwarkentin** [rocketmiles/aws-cct#1]
Very neat tool, this seems like it could be very helpful to generate some internal reports that I used to pull together through the Cost Explorer UI!

One of the things that I would include in the report would be a percentage delta for each service month to month, so for example:

```
| AWS CloudTrail                                  | $4,071.19   | $3,460.04   | -$611.15  |
```

Would include:

```
| AWS CloudTrail                                  | $4,071.19   | $3,460.04   | -$611.15  | -15% |
```

**Comment by rgoomar** [comment]
I'm glad this tool is helpful for you!

This is a great idea. It would also tie in well with the CSV feature.

**Comment by rgoomar** [comment]
Released in [v1.2.0](https://github.com/rocketmiles/aws-cct/releases/tag/v1.2.0)

**Comment by mwarkentin** [comment]
@rgoomar nice, thanks!
